Benchmarks/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿using BenchmarkDotNet.Running;
+using Benchmarks.Benchmarks;
+
+namespace Benchmarks
+{ 
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+             BenchmarkRunner.Run<ResponseBuffering>();
+        }
```

Comment:
Let's make this runnable for any future as well:
```suggestion
        public static void Main(string[] args) => BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args);
```